projects.json

```diff
@@ -828,8 +828,12 @@
     "url": "https://github.com/kickstarter/Kickstarter-Prelude",
     "path": "Kickstarter-Prelude",
     "branch": "master",
-    "maintainer": "stephen@stephencelis.com",
+    "maintainer": "justins@kickstarter.com",
     "compatibility": [
+      {
+        "version": "4.0",
+        "commit": "7b4eff0bae4f1abaf8ccc8370153f0945e0752ae"
+      },
       {
         "version": "3.0",
```

Comment:
Now that we have a 4.0 hashes, we can drop the 3.0 hashes. Would you mind updating this to remove the 3.0 configurations for Kickstarter-Prelude and Kickstarter-ReactiveExtensions? (That trailing comma after each 4.0 configuration also causes problems if it's not removed.)

Reply:
Will do!